### PR TITLE
Dns api regru

### PR DESCRIPTION
**Pull request by deflorator1980**

### CODE REVIEW

File: dnsapi/dns_regru.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  fi
+  _err "Could not create resource record, check logs"
+  _err "${response}"
+  return 1
+}
+
+dns_regru_rm() {
+  fulldomain=$1
+  txtvalue=$2
+  echo 'RM-ONE '"$1"
```
Comment (Neilpang) [inline dnsapi/dns_regru.sh]: do not use `echo`
please use `_info()`

**Comment by Neilpang** [comment]
please read this guide and check every item in it: 
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide#10-please-create-a-new-issue-for-future-bugs

**Comment by deflorator1980** [comment]
I did it. Is it ok? https://github.com/Neilpang/acme.sh/issues/2336:)

**Comment by Neilpang** [comment]
please add your usage here:  https://github.com/Neilpang/acme.sh/wiki/dnsapi
otherwise, nobody knows how to use it.

**Comment by deflorator1980** [comment]
> please add your usage here: https://github.com/Neilpang/acme.sh/wiki/dnsapi
> otherwise, nobody knows how to use it.

Done.

**Pull request merged**